knowledge-base/styling-tooltips-in-html5-based-report-viewers.md

```diff
@@ -106,12 +106,14 @@ You can provide styling for the tooltips in the **html**/**cshtml** page of your
 
 In case you need a custom solution for your tooltips, you can create your own ones, using the information provided in the **data-tooltip** attributes of the report viewer content. In the following example we will show you how to create tooltips using [jQueryUI Tooltip](https://jqueryui.com/tooltip/) widget:
     
- - First, declare the jQuery library in your **html** page:
+ - First, declare the __jQuery UI__ library in your **html** page. Here is a link to a CDN source:
 
 ```html
-<script src="http://code.jquery.com/ui/3.5.1/jquery-ui.min.js"></script>
+<script src="https://code.jquery.com/ui/1.13.0-rc.3/jquery-ui.min.js"></script>
 ```
 
+If the above link becomes non-functional, consider replacing it with a working one. Here is a link to the [jQuery CDN – Latest Stable Versions](https://code.jquery.com/).
```

Comment:
The URL should point to jQuery UI library to avoid confusion: https://jqueryui.com/ or https://releases.jquery.com/ui/